libs/market-list/src/lib/utils/market-list.utils.ts

```diff
@@ -20,20 +35,43 @@ export const mapDataToMarketList = ({ markets }: MarketList) =>
       )
       .map((m) => {
         return {
-          id: m.id,
-          decimalPlaces: m.decimalPlaces,
+          ...m,
           marketName: m.tradableInstrument.instrument?.code,
+          settlementAsset:
+            m.tradableInstrument.instrument.product?.settlementAsset?.symbol,
           lastPrice: lastPrice(m) ?? m.data?.markPrice,
           candles: (m.candles || []).filter((c) => c),
+          candleHigh: m.candles
+            ?.reduce(
```

Comment:
chore (blocking): This reduce is super gnarly and hard tor read. Can you refactor it into a function (`calcCandleHigh` maybe??), with less ternaries and just regular `if(){}` please

Reply:
PS. same for the candleLow property